.github/workflows/main.yml

```diff
@@ -518,3 +517,11 @@ jobs:
       if: startsWith(github.ref, 'refs/tags/')
       with:
         files: "gh-release/*.tar.gz*"
+    - uses: dtolnay/rust-toolchain@stable
+      if: startsWith(github.ref, 'refs/tags/')
+    - run: rustc publish.rs
+      if: startsWith(github.ref, 'refs/tags/')
+    - run: ./publish publish
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        CARGO_REGISTRY_TOKEN: ${{ secrets.CRATES_IO_TOKEN }}
```

Comment:
This was `CRATES_IO_TOKEN` before in `bump.yml`, which I think was incorrect.
I can't actually check the name of the secret, but worst case scenario we will find out when we try and publish.